chaos-bugbounty-list.json

```diff
@@ -7200,6 +7200,42 @@
         "sidefx.com"
       ]
     },
+    {
+      "name": "SlideShare",
+      "url": "https://app.inspectiv.com/#/programs/7AYpM2mTsFqUmbAzNEzapT",
+      "bounty": true,
+      "swag": false,
+      "domains": [
+        "slideshare.net"
+      ]
+    },
+    {
+      "name": "Scribd",
+      "url": "https://app.inspectiv.com/#/programs/Xf5bRH4ZxfjimoeRvAvCPc",
+      "bounty": true,
+      "swag": false,
+      "domains": [
+        "scribd.com"
+      ]
+    },
+    {
+      "name": "chewy",
+      "url": "https://app.inspectiv.com/#/programs/RBZwjFvUyBTqpkZBzcujph",
+      "bounty": true,
+      "swag": false,
+      "domains": [
+        "chewy.com"
+      ]
+    },
+    {
+      "name": "minehut",
+      "url": "https://app.inspectiv.com/#/programs/pDDJKtfaHLAT2AbGZZJFFb",
```

Comment:
@mdr3 looks like this URL isn't working for me - the others did once I signed into inspectiv...